tools/vmd/ios_vm_test_runner.template.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+# Copyright 2022 bazel-ios Authors. All rights reserved.
+#
+# This file patched to account for a VM base directory. Commonly in remote
```

Comment:
So IIUC [rules_apple's runner](https://github.dev/bazelbuild/rules_apple) script is being used as a base/reference and then changes are being made to make it work in a vm right? So I assume it is not the plan to eventually upstream any of this but instead continue to make it work better with the vm runner?

Reply:
Yeah, there is no plan to upstream the runner template. Mainly, it's a pretty small template and I imagine that it isn't worth the complexity or possible to have rules_apple to launching VMs. What I was thinking is to add the parameter to the Bazel rule